cpp/locality/NeighborList.cc

```diff
@@ -163,6 +163,18 @@ template unsigned int NeighborList::filter(bool*);
 
 unsigned int NeighborList::filter_r(float r_max, float r_min)
 {
+    if (r_max <= 0) 
+ { 
+     throw std::invalid_argument("RDF requires r_max to be positive."); 
```

Comment:
Update the names of the errors:
```suggestion
     throw std::invalid_argument("NeighborList.filter_r requires r_max to be positive."); 
```